news/2021-02-24-new-featured-artist-the-gentle-men.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "New Featured Artist: The Gentle Men"
+date: 2020-02-24 09:30:00 +0000
+---
+
+**The Gentle Men** waltz into the Featured Artist lineup!
+
+![](https://assets.ppy.sh/artists/142/header.jpg)
+
+Lyricist/vocalist Charlie White (a.k.a. Twitch streamer [MoistCr1TiKaL](https://www.twitch.tv/moistcr1tikal)) and seasoned musical expert Troy McKubre are **The Gentle Men** — a pair responsible for producing rock-focused audio experiences that blur the line between high effort jokes and legitimately good music.
+
+We managed to snag **all 6** of their currently released songs, each with a slightly different genre focus, like *Obsession*'s tilt towards electronic pop and *I Am Truth*'s metalcore tone.
+
+"Are these songs worth mapping?" you may be asking yourself. Charlie played through some maps of **The Gentle Men** songs recently, so if you're looking for a sample, give his video and these previews a peek:
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/YqEF_PHtEKw" frameborder="0" allowfullscreen></iframe>
```

Comment:
i think you meant to use `allowfullscreen="true"` here

Reply:
youtube embed has used name presence over value for several years at this point i believe

Reply:
current is correct according to other youtube embeds in news posts

Reply:
ah, i see